### PR TITLE
Add plotting tools

### PR DESCRIPTION
**Pull request by EmilyBourne**

### CODE REVIEW

File: plotting/plot_unstable.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+from fourier2d import Fourier2D
+
+from pygyro import splines as spl
+from pygyro.initialisation.constants import get_constants
+from pygyro.model.process_grid import compute_2d_process_grid
+from pygyro.model.grid import Grid
+from pygyro.model.layout import LayoutSwapper
+
+foldername = sys.argv[1]
+
```
Comment (FrederikSchnack) [inline plotting/plot_unstable.py:16]: Docstrings are missing

**Review by FrederikSchnack — CHANGES_REQUESTED** [review]
Apart from the comments it looks good to me

File: plotting/visualiseGrid.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+comm = MPI.COMM_WORLD
+rank = comm.Get_rank()
+
+if len(sys.argv == 1):
+    foldername = sys.argv[1]
+
+    distribFunc, constants, t = setupFromFile(foldername, comm=comm,
+                                              allocateSaveMemory=True,
+                                              timepoint=tEnd)
```
[the remark this answers is not in the document]
Comment (EmilyBourne) [reply plotting/visualiseGrid.py:24]: Fixed [79808e5](https://github.com/pyccel/pygyro/pull/31/commits/79808e5f777b4d9abec948d4f778499c09c55048)

File: plotting/visualiseGrid.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+from mpi4py import MPI
+import sys
+
+from pygyro.utilities.grid_plotter import SlicePlotterNd
+from pygyro.initialisation.setups import setupCylindricalGrid, setupFromFile
+
+comm = MPI.COMM_WORLD
+rank = comm.Get_rank()
+
+if len(sys.argv == 1):
```
[the remark this answers is not in the document]
Comment (EmilyBourne) [reply plotting/visualiseGrid.py]: Fixed [79808e5](https://github.com/pyccel/pygyro/pull/31/commits/79808e5f777b4d9abec948d4f778499c09c55048)

**Pull request merged**